plugins/rtmp-services/data/services.json

```diff
@@ -259,6 +259,40 @@
                 "max audio bitrate": 160
             }
         },
+	{
+            "name": "Loola.tv",
+            "common": false,
+            "servers": [             
+                {
+                    "name": "US East: Virginia",
+                    "url": "rtmp://rtmp.loola.tv/push"
+                },
+		{
+                    "name": "EU Central: Germany",
+                    "url": "rtmp://rtmp-eu.loola.tv/push"
+                },
+		{
+                    "name": "South America: Brazil",
+                    "url": "rtmp://rtmp-sa.loola.tv/push"
+                },
+		{
+                    "name": "Asia/Pacific: Singapore",
+                    "url": "rtmp://rtmp-sg.loola.tv/push"
+                },               
+                {
+                    "name": "Middle East: Bahrain",
+                    "url": "rtmp://rtmp-me.loola.tv/push"
+                }
+            ],
+            "recommended": {
+                "keyint": 2,
+                "profile": "main",
```

Comment:
What is your target audience? Main profile is only needed for 10+ year old devices and will reduce quality of the encoder.

Reply:
After running some tests I changed to "high" profile and removed the "zerolatency".
Thanks for the feedback!